test/move.js

```diff
@@ -0,0 +1,22 @@
+var R = require('../source');
+var eq = require('./shared/eq');
+
+var list = [8, 6, 7, 5, 3, 0, 9];
+
+describe.only('move', function() {
```

Comment:
Just a note to remove the `.only` before this gets merged :)

Reply:
Sorry, an oversight 😕